docs/1-Applying/resumes.md

```diff
@@ -78,24 +64,50 @@ Volunteer experience
 
+Similarily, show that you know programming languages by mentioning them in the work you've done with them and don't just list a bunch of them at the top of your resume. It's useful to have them all in one place of course. But as a general rule of thumb, any language/technology/acronym that you put in a list should also appear elsewhere on your resume.
 
-### On how old your points can be
+(There are some exceptions to this. For example, you might want to include Haskell to show that you like to challenge yourself intellectually, but your largest Haskell project is smaller than all your other projects. It's fine to include Haskell, as long as you are able to explain how you've used Haskell during an interview if asked. In general, you should be able to talk about anything on your resume during an interview).
 
-Once you're 2 years into university, I'd say don't talk about high school at all unless (1) you have very few things to say about your last two years or (2) your high school accomplishments are extremely impressive
+### Length
+
+One page is generally considered best, but I've had a two page resume in the past and know plenty of people that have done well with two page resumes. In general, I'd say if you have a two page resume, you should make sure it's okay if they only read the first page (i.e. the second page is like "bonus extra stuff about me").
+
+I like the idea of the one page resume because (1) people might not look at it long enough to even see the second page and (2) if you spread out your information accross two pages, you'll probably have less concentration of the good stuff. If you have a 2+ page resume, pick your best selling points accross those multiple pages and put just those on one page.
+
+In general, it might seem like writing more will make you seem more impressive, but if you write tons and tons only to fill space, it won't be impressive because it will be obvious that it's just filler. Short resumes clearly and concisely demonstrate your qualifications and are easier for a recruiter or potential employer to scan and understand. You do not want to dilute your impressive accomplishments in a sea of boring details!
 
+How do you know if you're being too verbose? If you do have a two-page resume, you should definitely at least do the exercise of shortening it until it fits into one page. Just think of it as a way to practice being concise, and ask your friends to cross out what they think is the least interesting if needed. You might not like the end result and decide not to keep it. That's fine. But you might also find out that it works just fine.
+
+It's useful to pay attention to the _information density_ of each of your sentences. How much information do I provide the reader? You can do this analysis on a word-by-word level. Here are a few examples inspired by resumes I've seen in real life.
```

Comment:
I'm a fan of separate pages esp since this file is pretty big - makes things more easy to digest I think? Like how small PRs are easier to review ^_^ (thanks!)